src/middlewares/passport.js

```diff
@@ -0,0 +1,5 @@
+import passport from "passport";
```

Comment:
This is a utility component, not a middleware. Move it to utils directory and call it in your controller